README.md

```diff
@@ -54,8 +63,8 @@ x = [[r_w1, ..., r_w25, g_w1, ..., g_wN, b_w1, ..., b_wN, n_w1, ..., n_wN]]
 
 ### Create the training data
 
-Training data should consist of two files per grid/year. One file is a polygon vector file (stored as a GeoPandas-compatible
-format like GeoPackage) of the training grid for a region. The other is a polygon vector file (stored in the same format)
+Training data pairs should consist of two files per grid/year. One file is a polygon vector file (stored as a GeoPandas-compatible
+format like GeoPackage) of the training grid for a region. The other file is a polygon vector file (stored in the same format)
```

Comment:
👍

Reply:
I think you can say checkout example in the data folder https://github.com/jgrss/cultionet/tree/main/data, right?